models/sd5/iris.sdf

```diff
@@ -0,0 +1,543 @@
+<sdf version='1.6'>
```

Comment:
Where is this different from the original IRIS? You know this is auto-generated by a xacro extension right?

Reply:
Oh sorry I specified the wrong target branch